tests/markers.py

```diff
@@ -0,0 +1,19 @@
+import docker
+import pytest
+from docker.errors import DockerException
+
+from moto import settings
+
+
+def _docker_is_available() -> bool:
+    try:
+        docker.from_env()
```

Comment:
Just a nitpick - I think we should import docker inside the try, and catch all exceptions. 
You may not even have the `docker`-module installed, if you're not going to run the tests anyway.

Reply:
Moved these more explicit bits to a dedicated test, so users are informed via a failing test that other tests may fail as well (and those tests are suppressed along with the switch).